examples/schemas/pos/PersonIndividualData.json

```diff
@@ -39,6 +39,14 @@
       "items": {
         "type": "string"
       }
+    },
+    "image": {
```

Comment:
Why 'image'. ?    
 This should be any binary and type should be something like 'binary', or 'base64'
 (and maybe an attribute for mime-type).